### DISCUSSION TITLE
Source for linux/ubuntu binaries?

### DISCUSSION THREAD

**Discussion by BobMCT** [debauchee/barrier#1367] [Q&A] [answered]
I cannot find the answer to this but can anyone point me/us to a source for compiled binaries for x86-64/AMD64 linux binaries?  I currently use v2.3.2 but have some issues that I'm hoping v2.4 might have addressed.

Thanks

**Comment by shymega** [comment, marked as answer]
It's not our job to supply x86/x86_64 Linux binaries - that's done by the distro. We supply source code for the upstream code itself - us - but not the downstream package. 

Even v2.4.0 wouldn't fix that.

On Tue, Nov 2, 2021, at 8:39 PM, Bob Mariotti wrote:
> I cannot find the answer to this but can anyone point me/us to a source 
> for compiled binaries for x86-64/AMD64 linux binaries? I currently use 
> v2.3.2 but have some issues that I'm hoping v2.4 might have addressed.
>
> Thanks
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/debauchee/barrier/discussions/1367>, or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAKF2QGBZW2K3D3IM4AA453UKBLAXANCNFSM5HHLYHHA>.

-- 
Kind regards,
--
Dom Rodriguez (also known as shymega)
